app/views/clients/hourly_events.erb

```diff
@@ -0,0 +1,11 @@
+<div class="container" align="center">
+  <h1>Hourly Event Breakdown</h1><br>
+    <table class="table table-striped table-narrow">
+      <% (1..24).to_a.each do |hour|%>
```

Comment:
this is too much logic in your views. your views should simply display the data passed from the controller (via variables). here, you're taking 1 - 24 and turning that into an array. this should be done somewhere else.